crates/houston/src/config.rs

```diff
@@ -32,12 +32,18 @@ impl Config {
                 // Win: C:\Users\Alice\AppData\Roaming\Apollo\Rover\config
                 // Mac: /Users/Alice/Library/Application Support/com.Apollo.Rover
                 ProjectDirs::from("com", "Apollo", "Rover")
-                    .ok_or(HoustonProblem::ConfigDirNotFound)?
+                    .ok_or(HoustonProblem::DefaultConfigDirNotFound)?
                     .config_dir()
                     .to_path_buf()
             }
         };
 
+        if fs::read_dir(&home).is_err() {
+            fs::create_dir_all(&home).map_err(|_| {
+                HoustonProblem::CouldNotCreateConfigHome(home.display().to_string())
+            })?;
+        }
+
```

Comment:
i added this so that we can catch errors related to finding the config directory a bit sooner, rather than having the errors be smothered up with a vague "no profiles found" for instance